js/widgets/accordion.js

```diff
@@ -31,7 +31,19 @@
 
 $.widget( "ui.accordion", $.ui.accordion, {
 	options: {
-		theme: null
+		theme: null,
+		icons: {
+			activeHeader: "ui-icon-caret-d",
+			header: "ui-icon-caret-r"
+		}
+	},
```

Comment:
As i know your already aware the accordions do not properly get height one page show to fix you need to add ( untested )
```js
_create: function() {
		this._super();

		// We need to refresh on page show so that the height can be calculated
		// when the widget is not hidden
		this._on( this.document, {
			"pagecontainershow": function( event, ui ) {
				if ( ui.toPage[ 0 ] === this.element.closest( ".ui-page" )[ 0 ] ) {
					this.refresh();
				}
			}
		} );
	},
```